client/notification.js

```diff
@@ -137,7 +135,20 @@ _.extend(NotifyClient.prototype, {
    */
   sendSms: function (templateId, phoneNumber, personalisation, reference) {
     return this.apiClient.post('/v2/notifications/sms',
-      createNotificationPayload('sms', templateId, phoneNumber, personalisation, reference));
+      createNotificationPayload('sms', templateId, String(phoneNumber), personalisation, reference));
```

Comment:
I would rather we throw an error if phone number is not a string. Casting it could lead to unexpected weird behaviour.

Reply:
ok